Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -199,6 +199,18 @@ class TelemetryWrapper {
         } else {
             GleanMetrics.Preferences.newTabExperience.set(NewTabAccessors.Default.rawValue)
         }
+        // Record `Home` setting, where Firefox Home is "Home", a custom URL is "other" and blank is "Blank".
+        let homePageSetting = NewTabAccessors.getHomePage(prefs)
+        switch homePageSetting {
+        case .topSites:
+            let firefoxHome = "Home"
+            GleanMetrics.Preferences.homePageSetting.set(firefoxHome)
```

Comment:
If we're only using the string here, I think it is okay just to put it in `.set()`

Reply:
(Sorry if I misunderstood!) I had to adjust it slightly here b/c Firefox Home returns "TopSites", a custom set home returns "HomePage" and blank returns "Blank." But what we want is: Firefox Home --> "Home", Custom Set Url --> "other" and Blank --> "Blank"